ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py

```diff
@@ -190,3 +190,32 @@ def test_paddle_asin(
         on_device=on_device,
         x=x[0],
     )
+
+# less_equal
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.less_equal",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+)
+
+def test_tensorflow_less_equal(
```

Comment:
Naming is not correct here 
```suggestion
def test_paddle_less_equal(
```